database-commons/src/test/java/io/cdap/plugin/db/CustomAssertions.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.db;
+
+import org.junit.Assert;
+
+import java.util.Arrays;
+
+/**
+ * Test util methods for custom assertions.
+ */
+public final class CustomAssertions {
+
+  /**
+   * The maximum delta between expected and actual floating point number for which both numbers are still considered
+   * equal.
+   */
+  public static final double DELTA = 0.01;
```

Comment:
this seems like a pretty large delta. The original calls look like they were much smaller.

Reply:
[Changed to:](https://github.com/data-integrations/database-plugins/pull/45/files#diff-8edf8ec3951da6baf44da046b785ee0fR32)
```
public static final double DELTA = 0.000001;
```